### DISCUSSION TITLE
关于我 - Eave Luo

### DISCUSSION THREAD

**Discussion by giscus** [EaveLuo/eave-web#3] [General]
# 关于我 - Eave Luo

Eave Luo's homepage, blog.

http://localhost:3000/about

**Comment by EaveLuo** [comment]
## comment test

👋👋👋

**Comment by XiangShanShanShan** [comment]
老师能和你打valorant吗

**Reply by EaveLuo** [reply]
可以呀，只是最近打cs多一点

**Reply by XiangShanShanShan** [reply]
哈哈哈考完和你约两把，你玩的我也都玩，备考没时间玩了

**Reply by EaveLuo** [reply]
先全力以赴备考，祝考上理想的院校哈

**Comment by parkgogogo** [comment]
请问下骆毅老师工作了几年了，太吊了 ～

写文章的经验能分享下嘛 ～

**Reply by EaveLuo** [reply]
哈哈哈编码6年多了~不算很有经验啦~写文章的话可以让agent订阅一些rss源和感兴趣的一些方向的官方，比如ai的我会关注御三家（openai、anthropic、google），国内的会看36氪和apposo，了解更多的信息，然后结合实际经验思考产出~

ps.内网同学为何不在内网交流哈哈哈哈跑我博客来了

**Reply by parkgogogo** [reply]
因为你太强了，bro

**Reply by EaveLuo** [reply]
bro过谦了，有想法可以一起交流哈~（不管是域内的还是域外的）
